src/Jackalope/ObjectManager.php

```diff
@@ -785,6 +801,7 @@ public function registerNodeTypesCnd($cnd, $allowUpdate)
                 $nsRegistry->registerNamespace($prefix, $uri);
             }
 
+            // TODO: Is @return type bool wrong?
```

Comment:
yes, the `@return` is wrong. this is called by NodeTypeManager::registerNodeTypesCnd which states its an \Iterator.

rather than put the whole description of that iterator here, we could do `@see NodeTypeManagerInterface::registerNodeTypesCnd`